modules/installation-azure-config-yaml.adoc

```diff
@@ -180,6 +180,11 @@ If you disable simultaneous multithreading, ensure that your capacity planning a
 <5> You can specify the size of the disk to use in GB. Minimum recommendation for control plane nodes is 1024 GB.
 //To configure faster storage for etcd, especially for larger clusters, set the
 //storage type as `io1` and set `iops` to `2000`.
++
+[NOTE]
+====
+Generation 2 VMs are supported and can be used to provision VMs with disk sizes that exceed 2000 GB. You must use a VM from a series which has link:https://docs.microsoft.com/en-us/azure/virtual-machines/generation-2#generation-2-vm-sizes[generation 2 support], such as `Standard_D2s_v3`.
```

Comment:
I'd suggest saying "Generation 2 VMs **in Azure** are supported..." since it seems to be specific to Azure VMs.